pkg/experiment/types.go

```diff
@@ -27,3 +29,12 @@ type Variant struct {
 	Key      string                 `json:"key,omitempty"`
 	Metadata map[string]interface{} `json:"metadata,omitempty"`
 }
+
+type FetchError struct {
+	StatusCode int
+	Message    string
+}
+
+func (e *FetchError) Error() string {
+	return fmt.Sprintf("Message: %s", e.Message)
+}
```

Comment:
What's this for? is this to implement the error interface? If so, golang error messages are almost always all lowercase :) minor nitpick.

Reply:
Yep, for the error interface!